cpp/daal/src/threading/export_lnxarm.ref.def

```diff
@@ -0,0 +1,64 @@
+;===============================================================================
+; Copyright 2023 Intel Corporation
```

Comment:
```suggestion
```

Reply:
Since it is an ARM specific file don't think that it woth keeping Intel in the header

Reply:
as mentioned above - this should be UXL copyright for new files.